### PR TITLE
Up Documentation DenyHosts for Debian Buster StableTitre

### PR DESCRIPTION
**Pull request by ZerooCool**
Up Documentation DenyHosts for Debian Buster StableTitre
Add a example for use a dev branch with git.

### CODE REVIEW

File: documentation/README-Installation-Debian-Buster.txt [hunk 1]
```diff
@@ -75,10 +75,13 @@ sudo apt-get install python3
 cd ~
 sudo git clone https://github.com/denyhosts/denyhosts.git
 
-# Using git allows developers and those wishing to test a new version, or, as a fix, to change the stable main branch to a developing branch.
-# It is at this point in the installation that you may need to change the branch on which you are positioned.
+# Using git allows developers and those wishing to test a new version, or, for fix a bug.
```
Comment (josev814) [inline documentation/README-Installation-Debian-Buster.txt:78]: Should update this to be 
```
# Using git allows developers and users the ability to test a new version or a bug fix.
```

File: documentation/README-Installation-Debian-Buster.txt [hunk 2]
```diff
@@ -75,10 +75,13 @@ sudo apt-get install python3
...
 # Currently and by default, you are using the stable master branch.
-# You can continue.
+# Change the stable main branch to a developing branch.
+# Exemple for test the branch for fix a new feature :
```
Comment (josev814) [inline documentation/README-Installation-Debian-Buster.txt:81]:
```
# The following is an example of using the code associated with a bug designated as bug_128 on github.
```

Comment (ZerooCool) [reply documentation/README-Installation-Debian-Buster.txt:81]: I look that. Ok.

Comment (ZerooCool) [reply documentation/README-Installation-Debian-Buster.txt]: Ok for me.

**Pull request merged**